src/workspacify.rs

```diff
@@ -0,0 +1,212 @@
+use anyhow::{anyhow, bail, ensure, Context, Result};
+use std::{
+    collections::HashMap,
+    env::current_dir,
+    fs::{self, OpenOptions},
+    io::Read,
+    path::{Path, PathBuf},
+    str::FromStr,
+};
+use structopt::StructOpt;
+use toml_edit::{value, Array, Document, Formatted, InlineTable, Item, KeyMut, Table, Value};
+use walkdir::WalkDir;
+
+const FILES_HAVE_PARENTS: &str = "This is a file. Every file has a parent; qed";
+
+#[derive(Debug, StructOpt)]
+pub struct Workspacify {
+    #[structopt(long)]
+    path: Option<PathBuf>,
```

Comment:
Some docs that this will be replaced by the working directory if not given.

These docs will be shown in the cli